src/framework/theme/components/chat/chat-message.component.ts

```diff
@@ -193,15 +202,34 @@ export class NbChatMessageComponent {
    */
   @Input() type: string;
 
+  @Input() customMessageData: any;
+
+  private readonly defaultMessageTypes: string[] = ['text', 'file', 'map', 'quote', undefined];
```

Comment:
Also, let's remove undefined from the array and handle it separately in the `isDefaultMessageType` method.